packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.5.0+20
```

Comment:
Same version note here.

Reply:
incremented the second version component